llvm/test/tools/llvm-readobj/ELF/Sparc/sparc-elf-headers.test

```diff
@@ -0,0 +1,57 @@
+# RUN: yaml2obj %s -o %t -DCLASS_NAME="ELFCLASS32" -DMACHINE_NAME="EM_SPARC" -DFLAG_NAME=""
```

Comment:
"sparc-" is an unnecessary prefix on this test name, since it's nested inside a "Sparc" directory.

It'd be nice having a brief overview comment at the top of this file describing the overall purpose of the test cases in this file.